pandas/tests/io/parser/c_parser_only.py

```diff
@@ -55,6 +56,13 @@ def test_buffer_rd_bytes(self):
             except Exception:
                 pass
 
+    def test_buffer_rd_bytes_bad_unicode(self):
+        # Regression test for #22748
+        b = BytesIO(b"\xB0")
+        t = TextIOWrapper(b, encoding='ascii', errors='surrogateescape')
+        with pytest.raises(UnicodeEncodeError):
+            pd.read_csv(t)
+
```

Comment:
What happens when you run this test against the Python engine?

Reply:
Hi @gfyoung 

I've moved this to common.py now. I guess that even if the interpreter crash is only reproducible in the c-parser, the parsers should behave the same.

Reply:
Yes, that is correct.  Rarely do we have tests that specific to a given parser, unless the parser does not support the behavior we are testing.